.github/workflows/main.yml

```diff
@@ -62,13 +66,16 @@ jobs:
       - name: Build and Test
         shell: bash
         run: |
-          choco install python --version 3.6.8
-          python -m pip install --upgrade pip
-          pip install awscli
           go install github.com/mitchellh/gox@latest
           $(go env GOPATH)/bin/ziti-ci generate-build-info common/version/info_generated.go version
-          CGO_ENABLED=true $(go env GOPATH)/bin/gox -os=windows -arch=amd64 -output=$GOX_OUTPUT ./...
-          aws s3 sync --no-progress release/ s3://ziti-cmd-build-tmp/${{ github.run_id }}
+          $(go env GOPATH)/bin/gox -cgo -os=windows -arch=amd64 -output=$GOX_OUTPUT ./...
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
```

Comment:
I did this for GRID too. It is also the only way to share "anything" between jobs. The solution is to serialize your data into another script or parsable data.

The most common use is for building artifacts though.